promoted-ruby-client.gemspec

```diff
@@ -37,9 +37,9 @@ Gem::Specification.new do |spec|
   spec.executables      = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths    = ["lib"]
 
-  spec.add_runtime_dependency 'faraday', '~> 1.4', '>= 1.4.3'
-  spec.add_runtime_dependency 'faraday_middleware', '~> 1.0', '>= 1.0.0'
-  spec.add_runtime_dependency 'faraday-net_http', '~> 1.0'
+  spec.add_runtime_dependency 'faraday', '>= 0.9.0'
+  spec.add_runtime_dependency 'faraday_middleware', '>= 0.9.0'
+  spec.add_runtime_dependency 'faraday-net_http', '>= 0.9.0'
```

Comment:
I suspect you don't need or want to update middleware and net http, they don't follow Faraday in lockstep

Reply:
I looked.  If faraday_middleware or faraday-net_http are around latest, they can't go before 1.0.